api/server/server.go

```diff
@@ -1206,7 +1220,10 @@ func postBuild(eng *engine.Engine, version version.Version, w http.ResponseWrite
 		authConfig        = &registry.AuthConfig{}
 		configFileEncoded = r.Header.Get("X-Registry-Config")
 		configFile        = &registry.ConfigFile{}
-		job               = eng.Job("build")
+		buildConfig       = builder.NewBuildConfig()
+		stdout            = engine.NewOutput()
+		stderr            = engine.NewOutput()
+		stdin             = engine.NewInput()
```

Comment:
I think `stdin` name here is little misleading. Because when now we removed jobs it is just `Context` or something like this.